### PR TITLE
Wizard: Add hostname to on-prem mapper (HMS-5183)

### PR DESCRIPTION
**Pull request by regexowl**
This adds hostname to on-prem mapper and updates a test that checks if the value got parsed correctly.<br/><br/>JIRA: [HMS-5183](https://issues.redhat.com/browse/HMS-5183)

### CODE REVIEW

File: src/Components/Blueprints/ImportBlueprintModal.test.tsx [hunk 1]
```diff
@@ -374,6 +374,9 @@ describe('Import modal', () => {
     );
     expect(keyboardDropDown).toHaveValue('us');
 
+    // Hostname
     await clickNext();
+    const hostnameInput = await screen.findByPlaceholderText(/Add a hostname/i);
+    expect(hostnameInput).toHaveValue('baseimage');
```
Comment (mgold1234) [inline src/Components/Blueprints/ImportBlueprintModal.test.tsx:380]: what do you think about 
```suggestion
    expect(hostnameInput).toHaveValue('base-image');
```

**Review by mgold1234 — DISMISSED** [review]
/lgtm

File: src/Components/Blueprints/ImportBlueprintModal.test.tsx (hunk 1 above)
Comment (regexowl) [reply src/Components/Blueprints/ImportBlueprintModal.test.tsx]: Sure, updated.

Comment (mgold1234) [reply src/Components/Blueprints/ImportBlueprintModal.test.tsx]: great, I think you can update this as well, it is not coming from that or, but for consistency :)
[customizations]
hostname = "baseimage"
fips = true

**Comment by mgold1234** [comment]
sorry, I saw you already updated it :)

**Review by mgold1234 — APPROVED** [review]
/lgtm

**Pull request merged**